src/main/java/com/prgrms/board/controller/PostController.java

```diff
@@ -0,0 +1,72 @@
+package com.prgrms.board.controller;
+
+import com.prgrms.board.dto.post.PostResponse;
+import com.prgrms.board.dto.post.PostSaveRequest;
+import com.prgrms.board.dto.post.PostUpdateRequest;
+import com.prgrms.board.dto.post.SimplePostResponse;
+import com.prgrms.board.service.PostService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/posts")
+public class PostController {
+
+    private final PostService postService;
+
+    @PostMapping
+    public ResponseEntity<PostResponse> createPost(
+            @RequestBody @Valid PostSaveRequest saveRequest
+    ) {
+        PostResponse postResponse = postService.createPost(saveRequest);
+        String resourceUrl = "/posts/" + postResponse.getPostId();
+        HttpHeaders headers = new HttpHeaders();
+        headers.setLocation(URI.create(resourceUrl));
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .headers(headers)
+                .body(postResponse);
+    }
+
+    @GetMapping
+    public ResponseEntity<List<SimplePostResponse>> getPagingPosts(
+            @RequestParam(defaultValue = "1") int page,
```

Comment:
```suggestion
            @PageableDefault(page = 1) Pageable pageable,
```
`Pageable`을 받아도 괜찮을 것 같아요~